### DISCUSSION TITLE
How to include a project in the list of plugins

### DISCUSSION THREAD

**Discussion by facundobatista** [pytest-dev/pytest#11066] [General]
Hello!

I see that [there is a list of plugins](https://docs.pytest.org/en/7.1.x/reference/plugin_list.html) which seems to be automatically populated from PyPI according to the project name.

However I have [a project that provides a simple but powerful way to assert logging](https://pypi.org/project/logassert/) which does not conform that name rule.

So, is there a way to have my project included in that list?

Thank you!!

**Comment by The-Compiler** [comment]
This currently seems to be [hardcoded in the script](https://github.com/pytest-dev/pytest/blob/dab199281c8f969ef870d842a660e40872785bd4/scripts/update-plugin-list.py#L58) - personally I wouldn't mind adding a `or match.groups()[0] in ADDITIONAL_PROJECTS` with a list of those defined above. Might want to make sense to open a PR with this?

**Comment by facundobatista** [comment]
Hello @The-Compiler, thanks for your comment!

I opened PR #11077 for this, following your idea (but refactoring the code a little to not make it repetitive).

Any feedback is welcomed! 

Thanks again,
